client-app/ui-kit/components/organisms/pagination/vc-pagination.stories.ts

```diff
@@ -0,0 +1,19 @@
+import { VcPagination } from "..";
```

Comment:
May be add all states (1,2,3,4,5, 1...4,5,6...10) to storybook to see all possible looks in UI in one place?